### ISSUE TITLE
Implement install_ansible_tower option to install or not install tower in a lab environment 

### ISSUE THREAD

**Issue by tima** [ansible/lightbulb#87]
[This comment refers to the refactored lab provisioner being developed in #81.]

Sometimes you are teaching how to install Tower, sometimes you are doing a demo where you just need it up and running. The lab provisioner should support a configuration option for both.

Default would be: no (don't install)

**Comment by samdoran** [comment]
I think rather than an option to install or not, it would be better to change the lab configuration to spin up the AMI with or without Tower. This goes along with notion of being able to specify a lab configuration via a single parameter, which will change which instances and how many are created.

**Comment by tima** [comment]
Yeah I guess this is more of a documentation issue than anything. At least initially. 

I think eventually the provisioner should have the option of pre-populating Tower for spinning up a quick demo environment for SAs and TAMs. It's probably worth considering a better means of communicating the default admin password to these users than SSH in and note the random password.
